app/main/views/suppliers.py

```diff
@@ -110,6 +110,43 @@ def edit_supplier(supplier=None, errors=None):
     )
 
 
+@main.route('/update/<int:id>', methods=['GET'])
+@main.route('/update/<int:id>/<path:step>', methods=['GET'])
+@main.route('/update/<int:id>/<path:step>/<path:substep>', methods=['GET'])
+@login_required
+def supplier_update(id=None, step=None, path=None):
+
+    try:
+        supplier = data_api_client.get_supplier(
+            current_user.supplier_code
+        )['supplier']
+    except APIError as e:
+        current_app.logger.error(e)
+        abort(e.status_code)
+
+    if feature.is_active('SELLER_UPDATE'):
+        # add business/authorized representative contact details
+        if len(supplier['contacts']) > 0:
+            supplier['email'] = supplier['contacts'][0]['email']
+            supplier['phone'] = supplier['contacts'][0]['phone']
+            supplier['representative'] = supplier['contacts'][0]['name']
+        if len(supplier['address']) > 0:
+            supplier['address']['address_line'] = supplier['address']['addressLine']
+            supplier['address']['postal_code'] = supplier['address']['postalCode']
```

Comment:
This props already exist in snake case on the address object returned for a supplier I thought?